PrimeNumbers/t_join.cpp

```diff
@@ -219,6 +316,7 @@ ulong t_join_mwaitx_loop_soft_wait_only::join(int inputIndex, int threadId, bool
     return totalIterations;
 }
 
+// This is no different from t_join_mwaitx_loop_soft_wait_only
```

Comment:
it is little different because there is a tight loop in `t_join_mwaitx_loop_soft_wait_only` that spins and checks for color between `monitorx` and `mwaitx`. In `t_join_mwaitx_noloop_soft_wait_only`, we don't check the color between the calls. Not sure if there is any visible difference though.

Reply:
I highly doubt we can detect that effect - that'd be one iteration of mwaitx saved. I didn't detect any diff at least but I also haven't run that many variations (if you were to detect any effect it would be when the workload is super balanced on each thread)